server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -71,10 +77,25 @@ func (r *Runner) Run(ctx workflow.Context, request Request) error {
 		return err
 	}
 
-	r.Scope.Counter("open_prs").Inc(int64(len(prs)))
-	if err := r.setRevision(ctx, request, prs); err != nil {
-		return errors.Wrap(err, "setting minimum revision for pr modifiying root")
+	version := workflow.GetVersion(ctx, version.MultiQueue, workflow.DefaultVersion, 1)
+	if version == workflow.DefaultVersion {
+		return r.setRevisionV1(ctx, request, prs)
 	}
+
+	oldPRs, newPRs := r.partitionPRsByDateModified(workflow.Now(ctx), prs)
```

Comment:
instead of partitioning, i'd rather we just sort it so that we're updating the most relevant PRs at a given time and then as you're iterating through the sorted list, just swap the tq when you've hit the threshold (instead of partitioning and batching like you're doing)